modules/apps/layout/layout-seo-web/src/main/java/com/liferay/layout/seo/web/internal/servlet/taglib/OpenGraphTopHeadDynamicInclude.java

```diff
@@ -332,6 +341,39 @@ private String _addLinkTag(LayoutSEOLink layoutSEOLink) {
 		return sb.toString();
 	}
 
+	private Set<Locale> _getAvailableLocales(
+			Layout layout, long plid, long siteGroupId)
+		throws JSONException {
+
+		Set<Locale> siteAvailableLocales = _language.getAvailableLocales(
+			siteGroupId);
+
+		if (!layout.isTypeContent()) {
+			return siteAvailableLocales;
```

Comment:
shouldn't this be `toLocaleSet(layout.getAvailableLanguageIds())`?

Reply:
thanks!

I talked with Pablo and yes, that will be the behavior.